zendesk/sunshine-conversations-api-quickstart-example
# Upgrades sunshine-conversations-client to 9.6.0

[Description and reason for the change](https://zendesk.atlassian.net/browse/SCE-1652)

This PR upgrades the `sunshine-conversations-client` package to 9.6.0 to fix a vulnerability found by Snyk.

Manual Tests:
1. Start the server by running the index.js
2. Start an ngrok tunnel to port 8000
3. Setup a webhook on an existing app that targets ngrok
4. Add a `conversation:message` trigger
5. Send a message

Outcome: The webhook payload should show up for the server running at 8000
